src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -899,9 +899,16 @@
         single id, <emphasis>anyone</emphasis>, that represents
         anyone.</para></listitem>
 
-        <listitem><para><emphasis role="bold">auth</emphasis> doesn't
-        use any id, represents any authenticated
-        user.</para></listitem>
+        <listitem><para><emphasis role="bold">auth</emphasis> is a special
+        scheme which ignores any provided ID and instead uses the current user,
+        credentials, and scheme. Any ID (whether, 'user' like with SASL
+        authentication or 'user:password' like with DIGEST authentication) provided is ignored
+        by the ZooKeeper server when persisting the ACL. However, the ID must be
+        provided in the ACL because the ACL must match the form 'scheme:id:perms'.
+        This scheme is provided as a convenience as it is a common use-case for
+        a client to create a znode and then restrict access to that znode to only that client.
```

Comment:
Avoiding the word "user" was intentional as a nod to some of the other auth schemes (e.g. the IP address one), but maybe that just creates more confusion than it's worth.